lib/Symbols.js

```diff
@@ -71,7 +101,17 @@ function isSymbolTerm(node, symbolName) {
       return true;
     }
   }
+  else if (hasDenominatorSymbol(node, symbolName)) {
```

Comment:
It's been a while now so I'm trying to remember.  I believe I had changed isSymbolTerm to return true if there was a symbol in the denominator, and there's logic that would then call getLastSymbolTerm.  And then getLastSymbolTerm relies on this isSymbolTerm function